CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/DataSources/HelpFormatsDataSource.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  HelpFormatsDataSource.swift
+//  
+//
+//  Created by Krzysztof Kryniecki on 03/08/2022.
+//  Copyright © 2022 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+public struct  HelpFormatsSection {
+    let items: [String]
+    let iconName: String
+}
+
+typealias HelpFormatsCollectionSection = (title: String,
+    items: [String],
+    itemsImage: UIImage?)
+
+class HelpFormatsDataSource: NSObject  {
+    let giniConfiguration: GiniConfiguration
+    
+    lazy var sections: [HelpFormatsCollectionSection] = {
+        var sections: [HelpFormatsCollectionSection] =  [
+            (NSLocalizedString(
```

Comment:
For all localizable resources, we need to use `NSLocalizedStringPreferredFormat` from GiniCaptureUtils. That will give the possibility to override localizable stings from the app bundle :).